src/main/kotlin/com/manimdsl/linearrepresentation/Objects.kt

```diff
@@ -0,0 +1,67 @@
+package com.manimdsl.linearrepresentation
+
+import com.manimdsl.shapes.Shape
+
+/** Objects **/
+
+interface Object : ManimInstr {
+    val ident: String
+}
+
+enum class ObjectSide(var coord: Pair<Double, Double>) {
+    ABOVE(Pair(0.0, 0.25)),
+    BELOW(Pair(-0.25, 0.0)),
+    LEFT(Pair(-0.25, 0.0)),
+    RIGHT(Pair(0.025, 0.0));
```

Comment:
0.25, 0.0

Reply:
@iz0825 ?

Reply:
Will address later

Reply:
I think clara is right i'll fix it and push again